libcloud/compute/drivers/ec2.py

```diff
@@ -3630,7 +3630,7 @@ def ex_register_image(self, name, description=None, architecture=None,
                           image_location=None, root_device_name=None,
                           block_device_mapping=None, kernel_id=None,
                           ramdisk_id=None, virtualization_type=None,
-                          ena_support=None):
+                          ena_support=None, billing_product=None):
```

Comment:
Shouldn't this be plural "billing_products"? Please also update the docstring as well.